scripts/PyChop/Chop.py

```diff
@@ -17,6 +17,7 @@
     http://www.neutronresearch.com/parch/1993/01/199301013280.pdf
 """
 
+from __future__ import division
```

Comment:
for all the files in the folder just add `from __future__ import (absolute_import, division, print_function)`. That is to prevent further modification to accidentally make the code python3 incompatible (like adding a print statement without parentheses)